### PR TITLE
Various updates for Ansible code best practices

### PR DESCRIPTION
**Pull request by jryburn**

### CODE REVIEW

File: plugins/modules/__init__.py
Comment (kentikethan) [inline plugins/modules/__init__.py:1]: you need the __init__.py file in order for the directory to be used as python modules. Necessary for some of the functions in the code and I believe necessary for ansible but not 100% on that. Did it pass ansible-test sanity on your local machine?

Comment (jryburn) [reply plugins/modules/__init__.py:1]: Added it back in.

**Comment by kentikethan** [comment]
looks good

**Review by kentikethan — APPROVED** [review]
looks good to me

**Pull request merged**